portfolio-project-matching-app/components/DBList.js

```diff
@@ -2,7 +2,7 @@ import DBListItem from './DBListItem'
 
 const DBList = ({ docList }) => {
```

Comment:
Similar to `DBForm.js` and `DBListItem.js`, doesn't really add anything, but provides a sandbox for seeing data from the DB.